packages/devextreme/scss/widgets/fluent/toast/_index.scss

```diff
@@ -31,16 +34,32 @@
 
 .dx-toast-info {
   background-color: $toast-info-bg;
+
+  &.dx-toast-content {
```

Comment:
I suggest we can't add additional specific to override color of text in this case. So, our styles can look as follows: 
```
.dx-toast-info {
    background-color: $toast-info-bg;
    color: $toast-info-color;
}
```